torch/nn/modules/module.py

```diff
@@ -476,7 +475,7 @@ def forward(self, x):
     call_super_init: bool = False
     _compiled_call_impl: Optional[Callable] = None
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self, *args: TypeVarTuple, **kwargs: Dict[str, Any]) -> None:
```

Comment:
This isn't right, you need to instationate a TypevarTuple as a TypeVar and then use it